minijinja/tests/snapshots/test_templates__vm@filters.txt.snap

```diff
@@ -1,9 +1,7 @@
 ---
 source: minijinja/tests/test_templates.rs
-assertion_line: 44
```

Comment:
I don't know why this changed

Reply:
I think your version of `cargo-insta` is probably too old.

Reply:
Actually in this case it's the other way round. Snapshots were made on an older insta version.